pkg/server/server.go

```diff
@@ -748,6 +763,33 @@ func (s *Server) indexHandler(w http.ResponseWriter, r *http.Request) {
 	if err := tpls.ExecuteTemplate(w, indexPageTemplateName, jsg); err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 	}
+
+	// This Content Security Policy (CSP) applies to Console web application resources.
+	// Console CSP is currently not enforced, but any violations are reported to Bridge.
+	cspDirectives := []string{
+		"default-src 'self'",
```

Comment:
this block should be moved above the ExecuteTemplate method execution
+ 
check if the `s.K8sMode` is set `off-cluster` or `on-cluster`